### PR TITLE
Create create-an-integration-detection-rule.md

### PR DESCRIPTION
**Pull request by eho1307**
### What does this PR do? What is the motivation?

Creating a new docs page for partners that outlines the steps required to package a SIEM detection rule OOTB with their integration.

### Merge instructions

Waiting on engineering to finish the feature. Will ping once this is ready to be merged. Please place it below the recommended monitors doc in the side nav. 

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by hestonhoffman** [comment]
Added [DOCS-6752](https://datadoghq.atlassian.net/browse/DOCS-6752) for someone on the docs team to help out

File: content/en/developers/integrations/create-an-integration-detection-rule.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+4. Save the JSON file and name it according to your detection rule title. For example, `your_integration_name_rule_name.json`.
+
+5. In the detection rule JSON file, add and fill out the `partnerRuleId`, and remove the `isEnabled` attribute. For more information, see [Configuration best practices](#configuration-best-practices).
+
+### Open a pull request
+
+1. Save the detection rule JSON file to your integration's `assets/security` folder.  
+
+2. Open a pull request (PR) to update the corresponding integration folder in either the [`integrations-extras` GitHub repository][5] or [`Marketplace` Github repository][6]. The PR should include your detection rule JSON file, along with any new integration files.
```
[the remark this answers is not in the document]
Comment (eho1307) [reply content/en/developers/integrations/create-an-integration-detection-rule.md:35]: Works for me!

File: content/en/developers/integrations/create-an-integration-detection-rule.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+[Datadog Cloud SIEM (Security Information and Event Management)][1] unifies developer, operation, and security teams through one platform. Datadog provides a set of out-of-the-box detection rules for many features and integrations. View these rules in your [SIEM Detection Rules list][2].
+
+Create an out-of-the-box detection rule to help users find value in your Datadog integration. This guide provides steps for creating an integration-detection rule and best practices to follow during the creation process.
+
+To create a Datadog integration, see [Create a New Integration][3].
+
+## Create a detection rule
+### Build a detection rule JSON schema
+
+1. Navigate to **[Security > Cloud SIEM > Detection Rules > New Rule][4]** and create a new rule.
```
Comment (clementgbcn) [inline content/en/developers/integrations/create-an-integration-detection-rule.md:21]: @nimishaaa we will have to tell partner that currently only threshold rule are supported.

File: content/en/developers/integrations/create-an-integration-detection-rule.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+### MITRE tags
+```
+The rule with partnerRuleId=<RULE_ID> contains a MITRE tag tactic but it does not contain the tag `security:attack` is not present in, please add it
+```
+When a rule contains a MITRE tag `tactic:<TAG_VALUE>`, the tag `security:attack` must be added to the list of tags.
+
+```
+The MITRE tactic/technique tag=<TAG> for partnerRuleId=<RULE_ID> appears to be incorrect (i.e. it does not exist in the MITRE framework).
+```
+The listed tactic/technique tag `<TAG>` does not follow the [MITRE framework](https://attack.mitre.org/). Please select a different tag.
```
Comment (clementgbcn) [inline content/en/developers/integrations/create-an-integration-detection-rule.md:105]: I think we should see `Please update the incorrect tag` instead.

File: content/en/developers/integrations/create-an-integration-detection-rule.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+```
+source=<SOURCE> in the tags of the rule with partnerRule=<RULE_ID> is not supported by Datadog documentation.
+```
+Reach out to Datadog to address the issue.
+
+### Rule content validation/ rule update
+```
+<RULE_ID_KEY>=<RULE_ID> name="<RULE_NAME>" - error=<ERROR>
+```
+Reach out to Datadog to address the issue.
```
Comment (clementgbcn) [inline content/en/developers/integrations/create-an-integration-detection-rule.md:128]: For this one, the error message should be sufficient to address the issue.

**Review by jhgilbert — APPROVED** [review]

File: content/en/developers/integrations/create-an-integration-detection-rule.md (hunk 4 above)
Comment (eho1307) [reply content/en/developers/integrations/create-an-integration-detection-rule.md:128]: To match the existing pattern, I'd like to include what they can do to address it.

File: content/en/developers/integrations/create-an-integration-detection-rule.md (hunk 3 above)
Comment (eho1307) [reply content/en/developers/integrations/create-an-integration-detection-rule.md:105]: Should we instead say 'Please select a valid MITRE tag'?

File: config/_default/menus/menus.en.yaml [hunk 5]
```diff
@@ -4322,14 +4322,18 @@ main:
     url: developers/integrations/create-an-integration-recommended-monitor
     parent: dev_tools_integrations
     weight: 408
+  - name: Create an Integration Detection Rule
```
Comment (nimishaaa) [inline config/_default/menus/menus.en.yaml:4325]: change title to: Create a Cloud SIEM Detection Rule

File: content/en/developers/integrations/create-an-integration-detection-rule.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
+---
+further_reading:
+- link: https://docs.datadoghq.com/security/cloud_siem/log_detection_rules
+  tag: Documentation
+  text: Log Detection Rules
+kind: documentation
+title: Create an Integration Detection Rule
```
Comment (nimishaaa) [inline content/en/developers/integrations/create-an-integration-detection-rule.md:7]: change title to: Create a Cloud SIEM Detection Rule

File: content/en/developers/integrations/create-an-integration-detection-rule.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+description: Learn how to create a detection rule for your integration.
```
Comment (nimishaaa) [inline content/en/developers/integrations/create-an-integration-detection-rule.md:8]: change to: Learn how to create a Cloud SIEM detection rule for your integration.

File: content/en/developers/integrations/create-an-integration-detection-rule.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+---
+## Overview
+
+[Datadog Cloud SIEM (Security Information and Event Management)][1] unifies developer, operation, and security teams through one platform. Datadog provides a set of out-of-the-box detection rules for many features and integrations. View these rules in your [SIEM Detection Rules list][2].
+
+Create an out-of-the-box detection rule to help users find value in your Datadog integration. This guide provides steps for creating an integration-detection rule and best practices to follow during the creation process.
```
Comment (nimishaaa) [inline content/en/developers/integrations/create-an-integration-detection-rule.md:14]: change to: Create an out-of-the-box detection rule to help users find security insights through your Datadog integration. This guide provides steps for creating an Cloud SIEM detection rule and best practices to follow during the creation process.

File: config/_default/menus/menus.en.yaml (hunk 5 above)
Comment (eho1307) [inline config/_default/menus/menus.en.yaml:4325]:
```suggestion
  - name: Create a Cloud SIEM Detection Rule 
  ```

**Pull request merged**